mkt/websites/models.py

```diff
@@ -49,6 +49,12 @@ def device_names(self):
         with no_translation():
             return [DEVICE_TYPES[d].api_name for d in device_ids]
 
+    def get_boost(self):
+        """
+        Returns the boost used in Elasticsearch for this website.
+        """
+        return 0.0
```

Comment:
The default should be 1.0, otherwise if you multiple the boost by the relevance you'll always get zero.